src/pgfimage.cpp

```diff
@@ -129,10 +129,10 @@ namespace Exiv2 {
 
         // And now, the most interresting, the user data byte array where metadata are stored as small image.
 
-        enforce(headerSize <= std::numeric_limits<uint32_t>::max() - 8, kerCorruptedMetadata);
+        enforce(headerSize <= std::numeric_limits<uint32_t>::max() - 8, kerTiffParsingError);
```

Comment:
Hum ... I am not 100% convinced about this change. I think the renaming makes sense in most of the places, but here I think it could be a bit confusing. In this point we are checking the PNG format according to its specification:

https://dev.exiv2.org/projects/exiv2/wiki/The_Metadata_in_PNG_files

and I think it would be confusing to see in the error message something about TIFF when reading the PNG metadata. 

Let's wait for some feedback from @D4N (or @clanmills) if possible.

Reply:
Note that I am aware that you are just following what it was written in #323 😉

Reply:
I am actually not sure whether we should replace all instances of `kerCorruptedMetadata` with `kerTiffParsingError` at all? When I created #323, we were using `kerCorruptedMetadata` very sparingly (afaik it had just been introduced, maybe even by me). However, with every fuzzed file, we just added a `if() { throw(kerCorruptedMetadata); }` and now a lot of these `throw`s & `enforce`s are not only for Tiff structures anymore, but general errors of invalid metadata.

All in all I think that we shouldn't do this renaming at all?

Reply:
I will revert the rename commit then for now.  I didn't know enough to understand these sections weren't dealing with Tiff but it did seem a bit strange.